src/context.rs

```diff
@@ -336,8 +336,24 @@ impl Game {
     #[cfg(feature = "save_state")]
     fn load_state(&mut self, state: MinimalState) {
         self.cpu.registers = state.cpu_regs;
-        todo!("load cpu io to cpu");
-        todo!("load cpu io to io_regs");
+
+        {
+            let cpu_io = Rc::new(RefCell::new(state.cpu_io_regs));
+            self.cpu.interrupt_flags = cpu_io.clone();
+            let io_bus = {
+                let mut builder = IORegBusBuilder::from(self.io_bus.take());
+                #[cfg(feature = "cgb")]
+                {
+                    builder.with_area(IORegArea::Key1, cpu_io.clone());
+                }
+                builder.with_area(IORegArea::IF, cpu_io.clone());
+                builder.build()
+            };
+            let io_bus = Rc::new(RefCell::new(io_bus));
+            self.addr_bus.io_reg = io_bus.clone();
+            self.addr_bus.ie_reg = cpu_io;
```

Comment:
`self.addr_bus.ie_reg = cpu_io.enable_mask` ?

Reply:
`cpu_io.enabled_mask` is an `u8`, `u8` doesn't implement the interface `FileOperation<..., Area>` but its parent structure does (`cpu_io` here)